File: src/main/resources/output/IFC4X3_csharp/Entities/IfcProductRepresentation/IfcMaterialDefinitionRepresentation/IfcMaterialDefinitionRepresentation.cs
public class IfcMaterialDefinitionRepresentation : IfcProductRepresentation
{
    public IfcMaterial RepresentedMaterial { get; set; }

    // === WHERE CLAUSES ===
    // OnlyStyledRepresentations : SIZEOF(QUERY(temp <* Representations | (NOT('IFC4X3_DEV_73740fe4.IFCSTYLEDREPRESENTATION' IN TYPEOF(temp))) )) = 0
}

File: src/main/resources/output/IFC4X3_vbnet/Entities/IfcProductRepresentation/IfcMaterialDefinitionRepresentation/IfcMaterialDefinitionRepresentation.vb
Public Class IfcMaterialDefinitionRepresentation Inherits IfcProductRepresentation
    Public RepresentedMaterial As IfcMaterial

    ' === WHERE CLAUSES ===
    ' OnlyStyledRepresentations : SIZEOF(QUERY(temp <* Representations | (NOT('IFC4X3_DEV_73740fe4.IFCSTYLEDREPRESENTATION' IN TYPEOF(temp))) )) = 0
End Class
